.github/workflows/update-wasm-tests.yml

```diff
@@ -0,0 +1,65 @@
+name: Update Wasm tests
+
+on:
+  # Trigger at every Sunday UTC noon, or manually.
+  schedule:
+    - cron: 0 12 * * 0
+  workflow_dispatch:
+
+jobs:
+  build-wpt:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout WPT repo
+        uses: actions/checkout@v4
+        with:
+          path: main
```

Comment:
Can we call this `wpt` or even `repos/wpt`? I found `main` confusing when I saw it later in the action and had to scroll back up to check it was actually the wpt root.

Reply:
Good point, done.